gallery/visualtests/nxButton.js

```diff
@@ -115,7 +174,8 @@ describe('NxButton', function() {
     };
 
     describe('Secondary styled button', function() {
-      const secondarySelector = `${selector} .nx-btn:not(.nx-btn--primary):not(.nx-btn--tertiary):not(.nx-btn--error)`;
+      const secondarySelector = `${selector} .nx-btn:not(.nx-btn--primary):not(.nx-btn--tertiary):not(.nx-btn--error)`
+      + ':not(.nx-btn--small)';
```

Comment:
I _think_ this can be simplified with the new-ish `:is` selector (which I've never used before, so I'm not certain):
```suggestion
      const secondarySelector = `${selector} .nx-btn:not(:is(.nx-btn--primary, .nx-btn--tertiary, .nx-btn--error, .nx-btn--small))';
```

Reply:
do you mean `.nx-btn:is` or `.nx-btn:not(:is..`

Reply:
> do you mean .nx-btn:is or .nx-btn:not(:is..

I meant not is.  We want the button that is _not_ primary, tertiary, error, or small.

Reply:
it works! 🎆  https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/d1edf6629c3d640d8f16dafce7bcbdacb0194ce0